src/components/MultiChoice/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import classNames from 'classnames';
+
+import Checkbox from 'components/Checkbox';
+import { Answer } from 'types/answer';
+
+export const multiChoiceDataTestIds = {
+  base: 'multi-choice__base',
+};
+
+interface MultiChoiceProps {
+  items: Answer[];
+  isPickOne: boolean;
+  onValuesChanged: (answers: Answer[]) => void;
+}
+
+const MultiChoice = ({ items, isPickOne, onValuesChanged }: MultiChoiceProps): JSX.Element => {
+  const [selectedValues, setSelectedValues] = useState<Answer[]>([]);
+
+  const isSelected = (answer: Answer): boolean => {
+    return !!selectedValues.find((item) => item.id === answer.id);
+  };
+
+  const getExtraClassNames = (answer: Answer): string => {
+    return isSelected(answer) ? '' : 'opacity-50';
+  };
+
+  const DEFAULT_TEXT_CLASSNAMES = 'text-white text-[20px] leading-[25px] tracking-survey-wider self-end';
+
+  const toggleCheckbox = (answer: Answer) => {
+    let newSelectedValues = [];
+    if (isSelected(answer)) {
+      newSelectedValues = selectedValues.filter((currentAnswer) => currentAnswer.id !== answer.id);
+    } else {
+      if (isPickOne) {
+        newSelectedValues = [answer];
+      } else {
+        newSelectedValues = [...selectedValues, answer];
+      }
+    }
+    setSelectedValues(newSelectedValues);
+    onValuesChanged(newSelectedValues);
```

Comment:
Please add a line break between blocks (declarations, conditions, actions,...)
```suggestion
    let newSelectedValues = [];

    if (isSelected(answer)) {
      newSelectedValues = selectedValues.filter((currentAnswer) => currentAnswer.id !== answer.id);
    } else {
      if (isPickOne) {
        newSelectedValues = [answer];
      } else {
        newSelectedValues = [...selectedValues, answer];
      }
    }

    setSelectedValues(newSelectedValues);
    onValuesChanged(newSelectedValues);
```